app/assets/javascripts/alert.js

```diff
@@ -0,0 +1,90 @@
+/* ==========================================================
+ * bootstrap-alert.js v2.0.4
+ * http://twitter.github.com/bootstrap/javascript.html#alerts
+ * ==========================================================
+ * Copyright 2012 Twitter, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ========================================================== */
+
+
+!function ($) {
+
+    "use strict"; // jshint ;_;
+  
+  
+   /* ALERT CLASS DEFINITION
+    * ====================== */
+  
+    var dismiss = '[data-dismiss="alert"]'
+      , Alert = function (el) {
```

Comment:
not necessary but look into arrow functions

Reply:
based on my limited understanding and this flow chart (linked in the article), https://github.com/getify/You-Dont-Know-JS/blob/master/es6%20%26%20beyond/fig1.png I'm going to not include the arrow functions in this code. However, in future content, i will look for more opportunities to use Arrow functions where appropriate, as they can make many chained functions much easier to read.